<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MyplexQA-PC" name="com.apalya.playcontent.tvshow_play_dtr" tests="1" failures="1" timestamp="5 Sep 2014 10:50:26 GMT" time="31.527" errors="0">
  <testcase name="tv_show_play" time="31.527" classname="com.apalya.playcontent.tvshow_play_dtr">
    <failure type="java.lang.RuntimeException" message="Exception caught while executing elementGetText: Failed to get text from &#039;//*[id=message]&#039;. Object repository doesn&#039;t contain: default\//*[id=message]">
      <![CDATA[java.lang.RuntimeException: Exception caught while executing elementGetText: Failed to get text from '//*[id=message]'. Object repository doesn't contain: default\//*[id=message]
	at com.experitest.client.Client.execute(Client.java:161)
	at com.experitest.client.Client.elementGetText(Client.java:542)
	at com.apalya.utils.keywords.get_text(keywords.java:184)
	at com.apalya.playcontent.tvshow_play_dtr.tv_show_play(tvshow_play_dtr.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- tv_show_play -->
</testsuite> <!-- com.apalya.playcontent.tvshow_play_dtr -->
